openid-4-verifiable-presentations-1_0.md

```diff
@@ -249,7 +249,7 @@ Depending on the Client Identifier Scheme, the Verifier can communicate a JSON o
 
 This specification enables the Verifier to send both Presentation Definition JSON object and Client Metadata JSON object by value or by reference.
 
-This specification defines the following new parameters:
+This specification defines the following new Authorization Request parameters:
```

Comment:
```suggestion
This specification defines the following new request parameters:
```

we have been avoiding to call this request authorization request because this can be a request for any grant type. would prefer we stick to that..

Reply:
this section is called `Authorization Request`(and has plenty more usage of the same language) should that change too?